openpype/hosts/max/plugins/publish/save_scenes_for_cameras.py

```diff
@@ -0,0 +1,101 @@
+import pyblish.api
+import os
+import sys
+import tempfile
+
+from pymxs import runtime as rt
+from openpype.lib import run_subprocess
+from openpype.hosts.max.api.lib_rendersettings import RenderSettings
+from openpype.hosts.max.api.lib_renderproducts import RenderProducts
+
+
+class SaveScenesForCamera(pyblish.api.InstancePlugin):
+    """Save scene files for multiple cameras without
+    editing the original scene before deadline submission
+
+    """
+
+    label = "Save Scene files for cameras"
+    order = pyblish.api.ExtractorOrder - 0.48
+    hosts = ["max"]
+    families = ["maxrender"]
+
+    def process(self, instance):
+        current_folder = rt.maxFilePath
+        current_filename = rt.maxFileName
+        current_filepath = os.path.join(current_folder, current_filename)
+        camera_scene_files = []
+        scripts = []
+        filename, ext = os.path.splitext(current_filename)
+        fmt = RenderProducts().image_format()
+        cameras = instance.data.get("cameras")
+        if not cameras:
+            return
+        new_folder = "{}_{}".format(current_folder, filename)
+        os.makedirs(new_folder, exist_ok=True)
+        for camera in cameras:
+            new_output = RenderSettings().get_batch_render_output(camera)       # noqa
+            new_output = new_output.replace("\\", "/")
+            new_filename = "{}_{}{}".format(
+                filename, camera, ext)
+            new_filepath = os.path.join(new_folder, new_filename)
+            new_filepath = new_filepath.replace("\\", "/")
+            camera_scene_files.append(new_filepath)
+            RenderSettings().batch_render_elements(camera)
+            rt.rendOutputFilename = new_output
+            rt.saveMaxFile(current_filepath)
+            script = ("""
+from pymxs import runtime as rt
+import os
+filename = "{filename}"
+new_filepath = "{new_filepath}"
+new_output = "{new_output}"
+camera = "{camera}"
+rt.rendOutputFilename = new_output
+directory = os.path.dirname(rt.rendOutputFilename)
+directory = os.path.join(directory, filename)
+render_elem = rt.maxOps.GetCurRenderElementMgr()
+render_elem_num = render_elem.NumRenderElements()
+if render_elem_num > 0:
+    ext = "{ext}"
+    for i in range(render_elem_num):
+        renderlayer_name = render_elem.GetRenderElement(i)
+        target, renderpass = str(renderlayer_name).split(":")
+        aov_name = directory + "_" + camera + "_" + renderpass + "." + "." + ext        # noqa
```

Comment:
With f-strings, it is much more readable. btw are the double dots before `{ext}` ok?
```suggestion
        aov_name = f"{directory}_{camera}_{renderpass}..{ext}"
```